src/providers/ad/ad_gpo.c

```diff
@@ -3849,7 +3850,13 @@ ad_gpo_sd_process_attrs(struct tevent_req *req,
     /* retrieve AD_AT_FUNC_VERSION */
     ret = sysdb_attrs_get_int32_t(result, AD_AT_FUNC_VERSION,
                                   &gp_gpo->gpo_func_version);
-    if (ret != EOK) {
+    if (ret == ENOENT) {
+        DEBUG(SSSDBG_IMPORTANT_INFO, "GPO with GUID %s is missing attribute "
```

Comment:
Please add the link to the MSDN documentation. Some parts are already quoted in ticket.
And Do we really need such hight level for debug message?
If I read ticket correctly it is not a critical problem.